src/common/Widget.js

```diff
@@ -688,6 +689,10 @@
 
     //  Render  ---
     Widget.prototype.render = function (callback) {
+        if (!this.visible() && (this._class.indexOf("common_Menu") === -1)) {
+            callback(this);
```

Comment:
@GordonSmith @jbrundage It turned out the reason that it was failing the unit tests is that the Menu widget initializes with _visible=false, which obviously causes problems here.  I put in a check to to see if it is a Menu widget and skip this block, but i feel this is kind of "hacky".  That said, I can't think of another way to go about this, and was wondering what your thoughts were.